app/components/ExamPages/SeventhChordsNotateTemplate.tsx

```diff
@@ -32,13 +37,16 @@ export default function NotateSeventhChords({
 
   useEffect(() => {
     currentUserDataRef.current = currentUserData;
+   // console.log("currentUserData: ", currentUserData);
   }, [currentUserData]);
 
   const handleSubmit = async (e: FormEvent) => {
     e.preventDefault();
     memoizedSetCurrentUserData({
       ...currentUserDataRef.current,
-      [seventhChordsPropName]: chords,
+      [seventhChordsPropName]: seventhChords,
+      [seventhChordDataPropName]: seventhChordData,
```

Comment:
We're getting errors here because there is no seventhChordData or seventhChordStaves